packages/devtools/test/composite.test.ts

```diff
@@ -520,7 +531,39 @@ describe('composite', () => {
     expect(params).not.toEqual(third)
   })
 
-  test.todo('Composite.create()')
+  describe('Composite.create()', () => {
+    test('creates a new composite from valid schema', async () => {
+      const composite = await Composite.create({
+        ceramic: ceramic,
+        schema: compositeSchemaWithProfiles,
+        metadata: {
+          controller: ceramic.did.id,
+        },
+      })
+      expect(composite.hash).not.toBeFalsy()
+      const compositeParams = composite.toParams()
+      expect(Object.keys(compositeParams.commits).length).toEqual(3)
+      expect(Object.keys(compositeParams.definition.models).length).toEqual(3)
+      const modelNames = ['GenericProfile', 'SocialProfile', 'PersonProfile']
+      Object.values(compositeParams.definition.models).map((modelDefinition: ModelDefinition) => {
```

Comment:
Not sure what you're trying to do here? Just check you have all the model names set?
If so, you could have something like:
```ts
const names = Object.values(compositeParams.definition.models).map(model => model.name)
for (const name of names) {
  expect(modelNames.includes(name)).toBe(true)
}
```

Reply:
I wanted to make sure that exactly the model names from the list are present, but I guess it's not super important.